src/java.base/share/classes/java/util/SortedMap.java

```diff
@@ -110,7 +110,7 @@
  * @since 1.2
  */
 
-public interface SortedMap<K,V> extends Map<K,V> {
+public interface SortedMap<K,V> extends SequencedMap<K,V> {
```

Comment:
This interface can now provide default implementations for the `firstKey` and `lastKey` methods:
```java
default K firstKey() {
	var entry = this.firstEntry();
	if (entry == null) {
		throw new NoSuchElementException();
	}
	return entry.getKey();
}
```
```java
default K lastKey() {
	var entry = this.lastEntry();
	if (entry == null) {
		throw new NoSuchElementException();
	}
	return entry.getKey();
}
```

Reply:
Doesn't seem all that useful.